src/avfoundation.cs

```diff
@@ -12461,6 +12726,21 @@ interface AVSampleBufferDisplayLayer {
 		[Notification]
 		NSString RequiresFlushToResumeDecodingDidChangeNotification { get; }
 
+		[TV (14, 5), Watch (7, 4), Mac (11, 3), iOS (14, 5)]
+		[Field ("AVSampleBufferDisplayLayerOutputObscuredDueToInsufficientExternalProtectionDidChangeNotification")]
+		[Notification]
+		NSString OutputObscuredDueToInsufficientExternalProtectionDidChangeNotification { get; }
+
+	}
+
+	[NoWatch]
+	[Category]
+	[BaseType (typeof(AVSampleBufferDisplayLayer))]
+	interface AVSampleBufferDisplayLayer_ProtectedContent
+	{
+		[TV (14, 5), Mac (11, 3), iOS (14, 5)]
```

Comment:
move to type level
it's not a big deal (only one member) and it's a category (not really used in code) but it's good practice for other cases :)